library/Icingadb/Widget/ItemList/StateListItem.php

```diff
@@ -137,4 +156,29 @@ protected function assemble(): void
             $this->createMain()
         ]);
     }
+
+    /**
+     * Shorten the given amount to 4 characters max
+     *
+     * @param int $amount
+     *
+     * @return string
+     */
+    protected function shortenAmount(int $amount): string
```

Comment:
You don't need this. As I discussed with @julianbrost already, Icinga 2 will not count more than 1000 (or so) objects. Just show the raw number and a plus afterwards if there are `1001`, so `1001 -> 1000+`, as there may actually be exactly 1000 objects. :roll_eyes: